Require Stream Analytics Query Tester role at subscription scope

Amitchauhangit opened the pull request:
To use this Process-data feature, Stream Analytics Query Tester role (https://www.azadvertizer.net/azrolesadvertizer/1ec5b3c1-b17e-4e25-8312-2acb3c3c5abf.html) is required at given subscription scope, which allows user to perform query testing without creating a stream analytics job first. Otherwise it throws unauthorized error. 

Example:
The client 'x@abc.com' with object id '<objectID>' does not have authorization to perform action 'Microsoft.StreamAnalytics/locations/sampleInput/action' over scope '/subscriptions/<subID>' or the scope is invalid. If access was recently granted, please refresh your credentials

Amitchauhangit committed:
Require Stream Analytics Query Tester role at subscription scope

To use this Process-data feature, Stream Analytics Query Tester role (https://www.azadvertizer.net/azrolesadvertizer/1ec5b3c1-b17e-4e25-8312-2acb3c3c5abf.html) is required at given subscription scope, which allows user to perform query testing without creating a stream analytics job first. Otherwise it throws unauthorized error. 

Example:
The client 'x@abc.com' with object id '<objectID>' does not have authorization to perform action 'Microsoft.StreamAnalytics/locations/sampleInput/action' over scope '/subscriptions/<subID>' or the scope is invalid. If access was recently granted, please refresh your credentials

spelluru reviewed (changes requested):
@Amitchauhangit - I have confirmed with the product team that what you observed and documented is correct. 

"User must be a member of the Stream Analytics Query Tester role at the Azure subscription level. This role allows user to perform query testing without creating a stream analytics job first."

Could you please update the statement and remove the link to the external site? I will merge this PR and then add the link to the section that the other content dev will be adding to the central RBAC article. Thanks.

Amitchauhangit committed:
Updated the link from our document.

Updated link from external site to the role in our documentation https://docs.microsoft.com/en-us/azure/role-based-access-control/built-in-roles#stream-analytics-query-tester

spelluru reviewed (commented):
Thanks for the update, but I have already made the change in an internal PR and have it merged. The update will go live around 3 PM PST today.
